src/main/java/org/symphonyoss/integration/webhook/jira/parser/v2/JiraMetadataParser.java

```diff
@@ -100,6 +132,32 @@ public JiraMetadataParser(UserService userService, IntegrationProperties integra
   }
 
   @Override
+  @PostConstruct
+  public void init() {
+    super.init();
```

Comment:
You should start this mapping into the constructor rather than override a callback function.